### PR TITLE
Added the testimonials to the readme file

### LINKED ISSUE #5019: [FEATURE] Include app testimonial in README
### Description

We have app testimonials on the homepage, but we should include them on the README as well.

I think these could be done in a table format

PR to add them to the homepage https://github.com/EddieHubCommunity/LinkFree/pull/5015

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by SohamRoy-01**
## Fixes Issue 
closes #5019

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (10)

Screenshot (11)

Screenshot (12)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5036"></a>

### CODE REVIEW

**Comment by SohamRoy-01** [comment]
### Hi @eddiejaoude, Hope you like my changes to the readme file..

**Review by Panquesito7 — COMMENTED** [review]
What if we add this in a separate `TESTIMONIALS.md` file? 🤔

**Comment by SohamRoy-01** [comment]
@Panquesito7 we can make a copy and to TESTIMONIALS.md file but as an issue addressed I did it accordingly..
Most people read the readme file to look into about the projects so I think it will be great to showcase

File: README.md [hunk 1]
```diff
@@ -90,6 +90,47 @@ Looking for inspiration? You can view the following profiles for an example:
 - [Krupali Trivedi || Chai](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/krupalitrivedi.json)
 - [Pradumna Saraf](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/Pradumnasaraf.json)
 
+# ⛩️ Testimonials
```
Comment (eddiejaoude) [inline README.md:93]: Each new section starts with `##`

File: README.md [hunk 2]
```diff
@@ -90,6 +90,47 @@ Looking for inspiration? You can view the following profiles for an example:
 - [Krupali Trivedi || Chai](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/krupalitrivedi.json)
 - [Pradumna Saraf](https://github.com/EddieHubCommunity/LinkFree/blob/main/data/Pradumnasaraf.json)
```
Comment (eddiejaoude) [inline README.md:91]: Now we have the testimonials, I think we can remove lines `85` to `91`

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍 a few small inline comments

File: README.md (hunk 2 above)
Comment (SohamRoy-01) [reply README.md:91]: Should I remove and Update it...

**Comment by eddiejaoude** [comment]
@SohamRoy-01 I just noticed this issue is not assigned to you and is now causing confusion. Please first request the issue to be assigned to you before doing any work on it - otherwise it creates more work for the community and maintainers

File: README.md (hunk 2 above)
Comment (eddiejaoude) [reply README.md:91]: yes please

File: README.md [hunk 3]
```diff
@@ -90,6 +90,62 @@ Looking for inspiration? You can view the following profiles for an example:
...
 
+
+<!-- Testimonials STARTs Here -->
+
+
+## ⛩️ Testimonials
+
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Here are some testimonials from individuals who have used LinkFree:<br><br>
```
Comment (eddiejaoude) [inline README.md:100]: why `<br>`?

Comment (eddiejaoude) [reply README.md:100]: why `&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`?

Comment (eddiejaoude) [reply README.md:100]: please just use standard markdown

Comment (SohamRoy-01) [reply README.md:100]: I used `<br>`  for inserts a single line break gives spaces for section and  `&nbsp;`  for  creating spaces in-front of line for alignment ...

Comment (Panquesito7) [reply README.md:100]: For single-line breaks, you can use `\` at the end of a sentence.

**Comment by SohamRoy-01** [comment]
I removed the `<br>` tag and  `&nbsp;`

**Review by Deva-1903 — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -82,13 +82,62 @@ This will allow you to run your favourite IDE but not have to install any depend
...
 
----
 
-Looking for inspiration? You can view the following profiles for an example:
+<!-- Testimonials STARTs Here -->
+
+
+## ⛩️ Testimonials
+
+
```
Comment (eddiejaoude) [inline README.md:91]: Double blank lines are ignored in markdown and only a single blank line is displayed, please use standard markdown, look at the rest of the file to be consistent

**Comment by eddiejaoude** [comment]
Thank you for making the changes @SohamRoy-01 , I made 1 more inline comment (which occurs in a few places)

**Comment by SohamRoy-01** [comment]
> Thank you for making the changes @SohamRoy-01 , I made 1 more inline comment (which occurs in a few places)

@eddiejaoude I changed the extra blank lines, Thank you for letting me do the task..

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by eddiejaoude** [comment]
Thank you for your efforts @SohamRoy-01 and making multiple changes

**Pull request merged**